src/app/views/sidebar/Sidebar.styles.ts

```diff
@@ -4,7 +4,7 @@ import {
 } from '@uifabric/styling';
 
 export const sidebarStyles = (theme: ITheme) => {
-  const pageHeight = '660px';
+  const pageHeight = '60vh';
```

Comment:
it's a little difficult to track which variable carry vh and which carry px so it may be useful to indicate it in the name.